docs/release/role-handbooks/README.md

```diff
@@ -0,0 +1,22 @@
+# role-handbooks
+
+These handbooks are maintained by current and previous contributors who have staffed these roles. They are intended to be living documents that evolve as the roles and project evolves. Do not treat them as rules set in stone, but guidelines to be re-examined.
+
+## Overview
```

Comment:
its good for context, but the hierarchy doesnt really make sense.  we have kind of an aside about role-handbooks as the main header, then the Overview as a subheader.  

the suggestion is to maintain the original `release-team-onboarding.md` in the parent folder with the overview so that it's an obvious entrypoint in the release docs.  then we can include this note on the role-handbooks as a sub-header in that doc, or just have it be on its own in this current doc